doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -458,8 +458,11 @@ Cellular samples (renamed from nRF9160 samples)
       This will fix a case where a battery-operated device joins a network that does not support PSM.
       This fulfills the proprietary PSM requirements of modem firmware v2.0.0.
 
-  * Updated the sample to use tickless operating mode from Zephyr's LwM2M engine, which does not cause device wake-up in 500 ms interval anymore.
-    This allows the device to achieve two µA of current usage while in PSM sleep mode.
+  * Updated:
+
+    * The sample to use tickless operating mode from Zephyr's LwM2M engine, which does not cause device wake-up in 500 ms interval anymore.
+      This allows the device to achieve two µA of current usage while in PSM sleep mode.
+    * Use CONFIG_LWM2M_UPDATE_PERIOD to configure the interval of sending LwM2M update.
```

Comment:
Isn't this something that was added (a new Kconfig option for configuring...)?

Reply:
No, this is not something that was added here. This is a Kconfig value that has been added in the Zephyr earlier. Now I'm just using it in the sample.